packages/rocketchat-ui-flextab/client/flexTabBar.js

```diff
@@ -188,15 +188,30 @@ Template.RoomsActionTab.helpers({
 		if (Template.instance().small.get()) {
 			return [];
 		}
+		let removedButton = false;
+		const canAddToChannel = RocketChat.authz.hasAllPermission('add-user-to-any-c-room');
+		const canAddToGroup = RocketChat.authz.hasAllPermission('add-user-to-any-p-room');
+		const canAddToJoinedRoom = RocketChat.authz.hasAllPermission('add-user-to-joined-room');
```

Comment:
Need to make sure the _room_ is created before he is able to `add-user-to-joined-room`.

Reply:
Yes. I think your comment on the createChannel.js file looks like you saw where that happens. I could take a look at how we use the add-user-to-joined-room permission. In this implementation this user must have both permissions 'add-user-to-any-c-room'('add-user-to-any-p-room' for groups) and 'add-user-to-joined-room' to view the 'add users' button. Perhaps it should only rely on the 'add-user-to-joined-room'? It is as simple as deleting the two const 'canAddToChannel' and 'canAddToGroup' then removing them from the if statement on line 202 and 206.

Reply:
Yep, checking the permissions page show that users do not have the `add-user-to-any-p-room` (same for `c`) by default, and only admins do. Also there could be cases where users don't have the `add-user-to-joined-room` permission (which would be weird, but could happen) and the former permission should overrule this one.
I'm doing some tests on the PR deployment to check for edge cases.